From ecb8724a853b176e942b40d9e95c950114cdb087 Mon Sep 17 00:00:00 2001
From: Chris Blake <chrisrblake93@gmail.com>
Date: Tue, 23 Jul 2019 13:44:29 -0500
Subject: [PATCH 12/21] arm64: dts: sun50i-h3-h5: Adjust OPP voltages

Minor adjustments since these are set high from the start.
---
 arch/arm/boot/dts/sunxi-h3-h5.dtsi | 26 +++++++++++++-------------
 1 file changed, 13 insertions(+), 13 deletions(-)

diff --git a/arch/arm/boot/dts/sunxi-h3-h5.dtsi b/arch/arm/boot/dts/sunxi-h3-h5.dtsi
index e5809642ba78..9a0e01f10c63 100644
--- a/arch/arm/boot/dts/sunxi-h3-h5.dtsi
+++ b/arch/arm/boot/dts/sunxi-h3-h5.dtsi
@@ -105,79 +105,79 @@
 
 		opp-120000000 {
 			opp-hz = /bits/ 64 <120000000>;
-			opp-microvolt = <1040000 1040000 1300000>;
+			opp-microvolt = <940000 940000 1300000>;
 			clock-latency-ns = <244144>; /* 8 32k periods */
 		};
 
 		opp-240000000 {
 			opp-hz = /bits/ 64 <240000000>;
-			opp-microvolt = <1040000 1040000 1300000>;
+			opp-microvolt = <940000 940000 1300000>;
 			clock-latency-ns = <244144>; /* 8 32k periods */
 		};
 
 		opp-480000000 {
 			opp-hz = /bits/ 64 <480000000>;
-			opp-microvolt = <1040000 1040000 1300000>;
+			opp-microvolt = <940000 940000 1300000>;
 			clock-latency-ns = <244144>; /* 8 32k periods */
 		};
 
 		opp-648000000 {
 			opp-hz = /bits/ 64 <648000000>;
-			opp-microvolt = <1040000 1040000 1300000>;
+			opp-microvolt = <960000 960000 1300000>;
 			clock-latency-ns = <244144>; /* 8 32k periods */
 		};
 
 		opp-816000000 {
 			opp-hz = /bits/ 64 <816000000>;
-			opp-microvolt = <1100000 1100000 1300000>;
+			opp-microvolt = <1000000 1000000 1300000>;
 			clock-latency-ns = <244144>; /* 8 32k periods */
 		};
 
 		opp-960000000 {
 			opp-hz = /bits/ 64 <960000000>;
-			opp-microvolt = <1200000 1200000 1300000>;
+			opp-microvolt = <1080000 1080000 1300000>;
 			clock-latency-ns = <244144>; /* 8 32k periods */
 		};
 
 		opp-1008000000 {
 			opp-hz = /bits/ 64 <1008000000>;
-			opp-microvolt = <1200000 1200000 1300000>;
+			opp-microvolt = <1100000 1100000 1300000>;
 			clock-latency-ns = <244144>; /* 8 32k periods */
 		};
 
 		opp-1056000000 {
 			opp-hz = /bits/ 64 <1056000000>;
-			opp-microvolt = <1320000 1320000 1320000>;
+			opp-microvolt = <1150000 1150000 1320000>;
 			clock-latency-ns = <244144>; /* 8 32k periods */
 		};
 
 		opp-1104000000 {
 			opp-hz = /bits/ 64 <1104000000>;
-			opp-microvolt = <1320000 1320000 1320000>;
+			opp-microvolt = <1170000 1170000 1320000>;
 			clock-latency-ns = <244144>; /* 8 32k periods */
 		};
 
 		opp-1152000000 {
 			opp-hz = /bits/ 64 <1152000000>;
-			opp-microvolt = <1320000 1320000 1320000>;
+			opp-microvolt = <1200000 1200000 1320000>;
 			clock-latency-ns = <244144>; /* 8 32k periods */
 		};
 
 		opp-1200000000 {
 			opp-hz = /bits/ 64 <1200000000>;
-			opp-microvolt = <1320000 1320000 1320000>;
+			opp-microvolt = <1240000 1240000 1320000>;
 			clock-latency-ns = <244144>; /* 8 32k periods */
 		};
 
 		opp-1224000000 {
 			opp-hz = /bits/ 64 <1224000000>;
-			opp-microvolt = <1340000 1340000 1340000>;
+			opp-microvolt = <1260000 1260000 1340000>;
 			clock-latency-ns = <244144>; /* 8 32k periods */
 		};
 
 		opp-1248000000 {
 			opp-hz = /bits/ 64 <1248000000>;
-			opp-microvolt = <1340000 1340000 1340000>;
+			opp-microvolt = <1280000 1280000 1340000>;
 			clock-latency-ns = <244144>; /* 8 32k periods */
 		};
 
-- 
2.17.1

